structured.js

```diff
@@ -478,7 +478,8 @@
      * peersToFind: The remaining ordered syntax nodes that we must find after
      *     toFind (and on the same level as toFind).
      */
-    function checkMatchTree(currTree, toFind, peersToFind, wVars, matchResults, options) {
+    function checkMatchTree(currTree, toFind, peersToFind, wVars, matchResults, options, recursing) {
```

Comment:
It looks like the jsdoc has gotten outdated for this function, can you update it?